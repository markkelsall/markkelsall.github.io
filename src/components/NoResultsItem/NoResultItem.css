.ui-state-disabled span {
    color: #989898;
    font-style: italic;
}
