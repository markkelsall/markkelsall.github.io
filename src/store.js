import {createStore, applyMiddleware, combineReducers} from 'redux';
import {composeWithDevTools} from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import pickupReducer from './reducers/pickUpReducer';

const reducer = combineReducers({
  pickUp: pickupReducer
})

export default createStore(
  reducer,
  composeWithDevTools(
    applyMiddleware(thunk)
  )
)