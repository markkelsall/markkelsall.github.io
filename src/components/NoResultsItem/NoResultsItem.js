import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './NoResultsItem.css';

export default class NoResultsItem extends Component {

    render = () => {
        return (
            <li className="ui-menu-item" role="menuitem" tabIndex="-1">
                <a className="ui-state-disabled">
                    <span>No results found</span>
                </a>
            </li>
        );
    }
}
