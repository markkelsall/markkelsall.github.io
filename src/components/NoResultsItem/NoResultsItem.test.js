import React from 'react'
import renderer from 'react-test-renderer'
import NoResultsItem from './NoResultsItem'
import { BrowserRouter as Router } from 'react-router-dom'
import { Provider } from 'react-redux'
import store from '../../store'

it ('Should render properly', () => {
    const component = renderer.create(
        <Provider store={store}><Router><NoResultsItem /></Router></Provider>,
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
})