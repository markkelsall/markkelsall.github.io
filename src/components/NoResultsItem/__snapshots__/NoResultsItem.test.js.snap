// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render properly 1`] = `
<li
  className="ui-menu-item"
  role="menuitem"
  tabIndex="-1"
>
  <a
    className="ui-state-disabled"
  >
    <span>
      No results found
    </span>
  </a>
</li>
`;
